### PR TITLE
Remove duplicated SVG dark files

### PR DESCRIPTION
**Pull request by Andreas-Kainz**
followup of PR #5067
cause there only duplicates from images was removed
not from images/dark

Signed-off-by: andreas kainz <kainz.a@gmail.com>
Change-Id: Ica099033c4be1fb2833509b5913abd9e6d49e9b2

### CODE REVIEW

**Comment by Andreas-Kainz** [comment]
@Minion3665 it's the follow up PR from #5067

File: browser/images/dark/lc_alignverticalcenter.svg [hunk 1]
```diff
@@ -1 +0,0 @@
... xmlns="http://www.w3.org/2000/svg"><path d="m7.99982 5.515625c-.0115-.454318.540283-.686847.857422-.361328l2.144531 2.148437-.0018-5.777343c-.017923-.350177.241538-.525391.500027-.525391s.51795.175214.5.525391l-.0022 5.777343 2.144531-2.148437c.317139-.325519.868926-.09299.857422.361328-.0041.130048-.05875.25337-.152344.34375l-2.912092 2.916016c-.160599.173289-.266379.223009-.435497.224609-.169118-.0016-.274898-.05132-.435547-.224609l-2.912109-2.916016c-.09359-.09038-.148244-.213702-.152344-.34375z" fill="#1e8bcd" fill-rule="evenodd"/><g fill="#fafafa" transform="scale(-1 1)"><rect height="1" ry=".5" width="18" x="-21" y="10"/><rect height="1" ry=".5" width="11" x="-14" y="13"/></g><path d="m7.99982 18.484375c-.0115.454318.540283.686847.857422.361328l2.144531-2.148437-.0018 5.777343c-.017923.350177.241538.525391.500027.525391s.51795-.175214.5-.525391l-.0022-5.777343 2.144531 2.148437c.317139.325519.868926.09299.857422-.361328-.0041-.130048-.05875-.25337-.152344-.34375l-2.912092-2.916016c-.160599-.173289-.266379-.223009-.435497-.224609-.169118.0016-.274898.05132-.435547.224609l-2.912109 2.916016c-.09359.09038-.148244.213702-.152344.34375z" fill="#1e8bcd" fill-rule="evenodd"/></svg>
```
Comment (Minion3665) [inline browser/images/dark/lc_alignverticalcenter.svg:1]: This SVG file doesn't appear to be mentioned in the aliases anywhere. Was it meant to be deleted?

File: browser/images/dark/lc_square.svg [hunk 2]
```diff
@@ -1 +0,0 @@
-<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="m2.5 2a.50005.50005 0 0 0 -.5.5v19a.50005.50005 0 0 0 .5.5h19a.50005.50005 0 0 0 .5-.5v-19a.50005.50005 0 0 0 -.5-.5zm.5 1h18v18h-18z" fill="#fafafa"/></svg>
```
Comment (Minion3665) [inline browser/images/dark/lc_square.svg:1]: This SVG file doesn't appear to be mentioned in the aliases anywhere. Was it meant to be deleted?

File: browser/images/lc_square.svg [hunk 3]
```diff
@@ -1,11 +0,0 @@
...
-<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
-  <path
-	 class="icn icn--shape-color" 
-     d="m2.5 2.5v19h19v-19z"
-     fill="#fafafa" 
-	 stroke="#3a3a38" 
-	 stroke-linecap="round" 
-	 stroke-linejoin="round"
-     />
-</svg>
```
Comment (Minion3665) [inline browser/images/lc_square.svg:11]: This SVG file doesn't appear to be mentioned in the aliases anywhere. Was it meant to be deleted?

**Review by Minion3665 — CHANGES_REQUESTED** [review]
Everything looks good apart from these 3 files which aren't mentioned at all in the aliases; I also haven't checked that the files you alias to actually exist, so someone else will need to do that

**Pull request closed without merging**